misc/services/camel/de-metas-camel-edi/src/main/resources/application.properties

```diff
@@ -45,11 +45,21 @@ edi.file.orders.ecosio=file://{{edi.local.dir}}/{{edi.file.orders.subdir}}/metas
 #edi.file.orders.ecosio=sftp://{{edi.sftpserver.ecosio.hostAndPort}}/{{edi.file.orders.subdir}}/metasfresh-xml
 
 edi.file.desadv.ecosio=file://{{edi.local.dir}}/{{edi.file.desadv.subdir}}/metasfresh-xml?charset={{edi.ecosio.charset.name}}
+#edi.file.desadv.in.house.v2=file://{{edi.local.dir}}/{{edi.file.desadv.subdir}}/metasfresh-xml?charset={{edi.ecosio.charset.name}}
 #edi.file.desadv.ecosio.remote=sftp://{{edi.sftpserver.ecosio.hostAndPort}}/{{edi.file.desadv.subdir}}/metasfresh-xml
+#edi.file.desadv.in.house.v2.remote=sftp://{{edi.sftpserver.ecosio.hostAndPort}}/{{edi.file.desadv.subdir}}/metasfresh-xml
```

Comment:
please generally call the propertiy `...inhouse...`. 
`in.house` might cause FUD due to the dedicated "in" ("is there also an `out.house`?" future-us will ask). 
AFAIU the env-variable then has to be called `INHOUSE` ( and can't be called `IN-HOUSE` or `IN_HOUSE`), but imo on balance thats preferable to `in.house`